### PR TITLE
Doc sweep fixes

### LINKED ISSUE #21834: Doc sweep
Review docs for best security practices.

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes the aspnetcore.docs issues for  #21834 
The aspnet issues will be in a seperate pr for that repo of course.

### CODE REVIEW

**Comment by wadepickett** [comment]
@Rick-Anderson, these are the aspnetcore.docs specific issues.  I noted in the PR  description above what was fixed or no longer a problem so no fix required.  These issues were raised in April 2020, several of the docs were updated since.

File: aspnetcore/security/anti-request-forgery.md [hunk 1]
```diff
@@ -430,7 +430,7 @@ xhttp.onreadystatechange = function() {
 xhttp.open('POST', '/api/password/changepassword', true);
 xhttp.setRequestHeader("Content-type", "application/json");
 xhttp.setRequestHeader("X-CSRF-TOKEN", csrfToken);
-xhttp.send(JSON.stringify({ "newPassword": "ReallySecurePassword999$$$" }));
+xhttp.send(JSON.stringify({ "newPassword": $CREDENTIAL_PLACEHOLDER$ }));
```
Comment (Rick-Anderson) [inline aspnetcore/security/anti-request-forgery.md:433]: @blowdart see just above [this bookmark](https://review.docs.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-6.0&branch=pr-en-us-21851#angularjs) for the preceding code.
Does this sample need to be about changing a pw?
 Can't we change the preceding code to:

```
xhttp.open('POST', '/api/token/changetoken', true);
xhttp.setRequestHeader("Content-type", "application/json");
xhttp.setRequestHeader("X-CSRF-TOKEN", csrfToken);
xhttp.send(JSON.stringify({ "newToken": $CREDENTIAL_PLACEHOLDER$ }));
```

Comment (Rick-Anderson) [reply aspnetcore/security/anti-request-forgery.md:433]: @wadepickett  if @blowdart  accepts this change, you'll need to replace all occurrences of `ChangePassword` to `ChangeToken`.

File: aspnetcore/security/data-protection/configuration/overview.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ public void ConfigureServices(IServiceCollection services)
 {
     services.AddDataProtection()
         .PersistKeysToFileSystem(new DirectoryInfo(@"\\server\share\directory\"))
-        .ProtectKeysWithCertificate("thumbprint");
+        .ProtectKeysWithCertificate($CREDENTIAL_PLACEHOLDER$);
```
Comment (Rick-Anderson) [inline aspnetcore/security/data-protection/configuration/overview.md:108]: You need to replace all occurrences of password in this document with a method secure method call.

Comment (Rick-Anderson) [reply aspnetcore/security/data-protection/configuration/overview.md:108]: > .ProtectKeysWithCertificate("thumbprint");

@blowdart  does `"thumbprint"` need to be replaced with `GetThumbprint()` ?

**Comment by Rick-Anderson** [comment]
@wadepickett  change all passwords to fetch that pw string from configuration.

File: aspnetcore/security/anti-request-forgery.md (hunk 1 above)
Comment (wadepickett) [reply aspnetcore/security/anti-request-forgery.md:433]: @blowdart , does the above pointed out by Rick look acceptable?

Comment (blowdart) [reply aspnetcore/security/anti-request-forgery.md:433]: Just changing to placeholder? Yup

File: aspnetcore/security/data-protection/configuration/overview.md (hunk 2 above)
Comment (blowdart) [reply aspnetcore/security/data-protection/configuration/overview.md]: This is removing thumbprint. Umm, it depends, If it's a thumbprint in config then yes, or you pass a cert directly

File: aspnetcore/security/anti-request-forgery.md (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/security/anti-request-forgery.md:433]: @blowdart, I want to remove password from the sample and replace it with `code`. The sample doesn't need to update a password.

Comment (blowdart) [reply aspnetcore/security/anti-request-forgery.md:433]: Fine by me then :)

File: aspnetcore/security/anti-request-forgery.md [hunk 3]
```diff
@@ -430,7 +430,7 @@ xhttp.onreadystatechange = function() {
 xhttp.open('POST', '/api/password/changepassword', true);
```
Comment (Rick-Anderson) [inline aspnetcore/security/anti-request-forgery.md:430]:
```suggestion
xhttp.open('POST', '/api/token/changeCode', true);```

Comment (wadepickett) [reply aspnetcore/security/anti-request-forgery.md]: applied  locally and pushed.

**Comment by wadepickett** [comment]
@Rick-Anderson, I also looked through the rest of the  two sample apps related to anti-request-forgery.md to see if there was anything else to change there.  There wasn't.   Ready for review.  Thanks for all your help.

File: aspnetcore/security/anti-request-forgery.md [hunk 4]
```diff
@@ -427,10 +427,10 @@ xhttp.onreadystatechange = function() {
         }
     }
 };
-xhttp.open('POST', '/api/password/changepassword', true);
+xhttp.open('POST', '/api/token/changeCode', true);
 xhttp.setRequestHeader("Content-type", "application/json");
 xhttp.setRequestHeader("X-CSRF-TOKEN", csrfToken);
-xhttp.send(JSON.stringify({ "newPassword": "ReallySecurePassword999$$$" }));
+xhttp.send(JSON.stringify({ "newCode": $CREDENTIAL_PLACEHOLDER$ }));
```
Comment (Rick-Anderson) [inline aspnetcore/security/anti-request-forgery.md:433]: OK for short term band aid, but  #21872 is the right way to fix this.

Comment (wadepickett) [reply aspnetcore/security/anti-request-forgery.md:433]: @Rick-Anderson, it is not clear to me if you want this to merge now and follow  #21872 for an update later, or if this PR should wait.

**Comment by Rick-Anderson** [comment]
@wadepickett  please add links to internal review

**Pull request merged**